primer_designer/views.py

```diff
@@ -170,7 +164,15 @@ def create(request, regions_form):
 
     # unique name of date and random 5 char str
     outname = f'{time_stamp()}-{random_string()}'
-    out_dir = f'{Path(__file__).parent.parent.absolute()}/static/tmp/{outname}/'
+
+    # making an output directory in /primer_designer for output PDF
+    parent_path = Path(__file__).parent.parent.absolute()
+    Path(f'{parent_path}/primer_designer/output/').mkdir(
```

Comment:
is this creating the dir in the primer designer container?

Reply:
removed this line and shifted the generated PDFs output straight to /static/tmp/{output_dir}